js/sts-hotspot.js

```diff
@@ -0,0 +1,60 @@
+AFRAME.registerComponent('sts-hotspot', {
+  schema: {
+    show: { type: 'selector' },
+    sound: { type: 'selector' },
+    playing: { default: false }
+  },
+
+  init: function () {
+    var self = this;
+    var popupEl = this.data.show;
+
+    popupEl.dataset.hotspot = self.el.id;
+
+    self.el.addEventListener('mouseenter', function () {
+      self.playHotspotSound();
+
+      popupEl.setAttribute('material', { visible: true });
+      popupEl.setAttribute('visible', true);
+    });
+  },
+
+  playHotspotSound: function () {
+    this.el.setAttribute('sts-hotspot', { playing: true });
+    this.el.dataset.playing = true;
+  },
+
+  stopHotspotSound: function () {
+    this.el.setAttribute('sts-hotspot', { playing: false });
+    this.el.removeAttribute('data-playing');
+  },
+
+  stopAllHotspotSound: function () {
+    var AllHotspotSound = document.querySelectorAll('[data-playing]');
+    AllHotspotSound.forEach(function (hotspotEl) {
```

Comment:
to be safe:

```js
Array.prototype.forEach.call(document.querySelectorAll('a-scene [data-playing]'), function (hotspotEl) {
  hotspotEl.setAttribute('sts-hotspot', { playing: false });
});
```

Reply:
nit: thinking about this some more: could we instead call `hotspotEl.components['sts-hotspot'].stopSound()` on each? would be more DRY, right?

Reply:
Yes, `Nodelist.forEach` is supported for most everything except IE, but yeah I'll use `Array.forEach` since we want this to work for Edge VR too.

Regarding stopSound, the general approach with A-Frame is to set props.  I could also use `hotspotEl.setAttribute('sts-hotspot', 'playing', false')`